app/controllers/api/ims_exports_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Api::ImsExportsController < ApplicationController
+
+  include Concerns::CanvasImsccSupport
+
+  def show
+    @export = ImsExport.find_by(token: params[:id])
```

Comment:
If this only responds with json, there's probably no need to do @export as there's no view.

Reply:
Done